packages/geolocation/src/drivers/mongodb/connection.js

```diff
@@ -17,18 +17,27 @@ const client = new MongoClient(uri);
  * const places = collection.find()
  */
 
```

Comment:
Hey guys remember to change the way you are importing Logger utility

a) if you need to use Logger const { Logger } = require('@booking-services/shared');
b) if you need to reference logger file @booking-services/shared/src/utils/logger

Reply:
Hello Hector!
I did it! Thanks for the advice! Please, could you review the changes to merge the pr?